dev_tools/conf/pip-list-dev-tools.txt

```diff
@@ -39,5 +39,8 @@ ipykernel
 # For verifying rst
 rstcheck~=3.3.1
 
+# for verifying github's code owners file
+codeowners
+
```

Comment:
This is enough to make the devtools themselves happy, but testing the devtools probably requires this to be added to `requirements.txt` as well.

Reply:
No, requirements.txt should never have testing related deps. 
The issue is something weird with the codeowners lib on Mac (weirdly because it installs locally just fine for me) and Windows (which I'll test in a sec on my Win box).

Reply:
It turns out Win wheels are not published at all (https://github.com/sbdchd/codeowners/issues/5) and it seems not all Mac wheels are published either. We should write our own parser eventually, but for now this works. I added a comment and turned off the new tests on win / mac.

Reply:
oh I'm silly, I need to somehow pip install only on linux! I'll do that soon too.